contracts/PartyBid.sol

```diff
@@ -346,27 +346,51 @@ contract PartyBid is ReentrancyGuardUpgradeable, ERC721HolderUpgradeable {
         );
     }
 
-    // ======== External: Recover =========
+    // ======== External: Emergency Escape Hatches (PartyDAO Multisig Only) =========
 
     /**
-     * @notice If the NFT gets stuck in the PartyBid
-     * (e.g. because of a faulty MarketWrapper that marks the auction Lost)
-     * the PartyDAO Multisig can transfer the NFT to the multisig
+     * @notice Escape hatch: in case of emergency,
+     * PartyDAO can use emergencyWithdrawEth to withdraw
+     * ETH stuck in the contract
      */
-    function recover() external {
+    function emergencyWithdrawEth(uint256 _value) external {
         require(
             msg.sender == partyDAOMultisig,
```

Comment:
not a big deal but would be nice to have an `onlyMultisig` modifier and use it here and `emergencyForceLost`